SU-HSDO/suhumsci
# SHS-5808: Uninstall Webform module

# READY FOR REVIEW

## Summary
- Removes config for `hs_webform` module, paragraphs, and related field config
- Removes `hs_webform` dependencies from config ignored configs
- Uninstalls the `hs_webform` module
- Removes config for `webform` and `webform_ui` modules
- Uninstalls `webform` and `webform_ui` modules
- Does not remove `hs_webform` or `webform` from the filesystem - to be done in a subsequent PR

## Need Review By (Date)
2024-09-23

## Urgency
low

## Steps to Test
1. Login to the site as an admin
2. Visit the paragraph types page `/admin/structure/paragraphs_type` and verify that there is no longer a "Webform" paragraph type (`hs_webform`)
3. Visit the modules listing page `/admin/modules` and verify that the following modules are now uninstalled: `hs_webform`, `webform_ui` and `webform`.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208357464145158
